src/Umbraco.Web.UI.Client/src/packages/user/current-user/types.ts

```diff
@@ -30,6 +30,7 @@ export interface UmbCurrentUserModel {
 	permissions: Array<DocumentPermissionPresentationModel | UnknownTypePermissionPresentationModel>;
 	unique: string;
 	userName: string;
+	userGroupIds: string[];
```

Comment:
Please use `unique` instead of `id`

Reply:
Hi Mads, I agree we should do that. Unfortunately that would be a breaking change as it is already called UserGroupIds on the regular user model. I'll put in a ticket for it for Umbraco 16.

Reply:
Oh, we can of course add it as an extra property in our end.